pkg/proxy/proxy.go

```diff
@@ -30,17 +30,21 @@ var (
 	ErrFileToStringEmpty error = errors.New("file is empty")
 )
 
+type loadTestSpecCreator func(*http.Request, *zap.Logger) (apisLoadTestV1.LoadTestSpec, error)
```

Comment:
```suggestion
var fromHTTPToSpecCreatorFn = fromHTTPRequestToLoadTestSpec
```
then use it as I showed below

Reply:
package-level vars is bad idea in general, explicit deps injection is much more straightforward

Reply:
It doesn't have any shared state to be bad, just another way of dealing with dependencies.

Reply:
Just a legal way of monkey-patching for tests 😜